miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultExperimentService.java

```diff
@@ -189,4 +207,18 @@ public void setStudyService(StudyService studyService) {
     this.studyService = studyService;
   }
 
+  public void setLibraryService(LibraryService libraryService) {
+    this.libraryService = libraryService;
+  }
+
+  @Override
+  public Collection<Experiment> listAllByLibraryId(long id) throws AuthorizationException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'uk.ac.bbsrc.tgac.miso.service.security.AuthorizationException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)